src/Xamarin.Android.Tools.Bytecode/ClassPath.cs

```diff
@@ -240,6 +240,15 @@ JavaDocletType GetDocletType (string path)
 				kind = JavaDocletType.Java7;
 			else if (rawHTML.Contains ("Generated by javadoc (1.8"))
 				kind = JavaDocletType.Java8;
+
+			// Check to see if it's an api.xml formatted doc
+			buf = new char[buf.Length];
+			using (var reader = File.OpenText(path))
```

Comment:
We use Mono's coding convention, so please use a space ` ` before `(` here and elsewhere.